CMakeLists.txt

```
@@ -29,14 +29,20 @@ include( "demos/logging-stack/logging.cmake" )
 
 # Configure options to always show in CMake GUI.
 option( BUILD_TESTS
-        "Set this to ON to build both demo and test executables. When OFF, only demo executables are built."
+        "Set this to ON to build test executables."
         OFF )
+option( BUILD_DEMOS
+        "Set this to ON to build demo executables."
+        ON )
 option( BUILD_CLONE_SUBMODULES
         "Set this to ON to automatically clone any required Git submodules. When OFF, submodules must be manually cloned."
         ON )
 option( DOWNLOAD_CERTS
         "Set this to ON to automatically download certificates needed to run the demo. When OFF, certificates must be manually downloaded."
         ON )
+option( INSTALL_PLATFORM
```

Pedantic suggestion for better readability
```suggestion
option( INSTALL_PLATFORM_LIBS
```